keanu-project/src/main/java/io/improbable/keanu/tensor/KeanuValueException.java

```diff
@@ -0,0 +1,16 @@
+package io.improbable.keanu.tensor;
+
+import io.improbable.keanu.tensor.bool.BooleanTensor;
+
+public class KeanuValueException extends RuntimeException {
```

Comment:
Maybe not the best name... it means there's an error relating to a value in the tensor, except it might also be in a scalar so I thought it wouldn't be right to call it TensorValueException.

Suggestions for a better name welcome!

Reply:
I've renamed it `TensorValueException`.